### ISSUE TITLE
Why pyexcel leaves always a .bak file during an ods. file creation ?

### ISSUE THREAD

**Issue by Gryffoon** [pyexcel/pyexcel#34]
Why pyexcel leaves always a .bak file during an ods. file creation ?

![2016-04-21_105605](https://cloud.githubusercontent.com/assets/15887222/14703425/0e417370-07b0-11e6-8f15-ee98b44c589b.png)

This file stays even if the python script execution is finished. 

When I open the .bak file with excel I obtain exactly the same file than the .ods one

**Comment by chfw** [comment]
This is [the behavior from ezodf](https://github.com/T0ha/ezodf/blob/master/ezodf/document.py#L114) and [backup is created by default](https://github.com/T0ha/ezodf/blob/master/ezodf/document.py#L190). I didn't pay much attention to this before but can make `backup=False` by default in pyexcel-ods3.

**Comment by Gryffoon** [comment]
Ok good thing to do for the next version

**Comment by chfw** [comment]
Please try out the fix if you have time:

    pip install https://github.com/pyexcel/pyexcel-ods3/archive/master.zip

The fix is pending for release.

**Comment by Gryffoon** [comment]
This version does not work at all:

```
>>> from pyexcel_ods3 import get_data, save_data
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\Snake\AppData\Local\Programs\Python\Python35\lib\site-packages\pyexcel_ods3\__init__.py", line 15, in <module>
    from pyexcel_io.io import get_data as read_data, isstream, store_data as write_data
ImportError: No module named 'pyexcel_io.io'
```

With the previous version:

```
>>> from pyexcel_ods3 import get_data, save_data
>>> 
```
Everything was good

**Comment by chfw** [comment]
Hmm, I think you will need newer but un-released version of pyexcel-io:

    pip install https://github.com/pyexcel/pyexcel-io/archive/master.zip

**Comment by chfw** [comment]
I will update its dependecy requirements.

**Comment by chfw** [comment]
released in pyexcel-ods3 v0.2.0
